### PR TITLE
rename stack

### PR DESCRIPTION
**Pull request by artemanufrij**
Signed-off-by: Artem Anufrij <artem.anufrij@live.de>

### CODE REVIEW

**Comment by artemanufrij** [comment]
Fix for: #135

**Comment by pixelipo** [comment]
works, apart from a small bug - it's impossible to unfocus the input field if you don't make any changes.

**Comment by artemanufrij** [comment]
@pixelipo fixed

**Comment by pixelipo** [comment]
@artemanufrij fixes first part, but second one is still an issue. Please add this:

```
.stack h2 span {
    max-width: 100%;
    white-space: normal;
    overflow: hidden;
    text-overflow: ellipsis;
}
```

**Comment by artemanufrij** [comment]
@pixelipo done

**Comment by artemanufrij** [comment]
@pixelipo pls entry `#nextcloud-deck` on `freenode` :)

File: css/style.css [hunk 1]
```diff
@@ -267,6 +271,8 @@ button:hover {
 .stack h2 span {
 	width: 100%;
 	white-space: normal;
+	overflow: hidden;
+    text-overflow: ellipsis;
```
Comment (juliushaertl) [inline css/style.css]: indentation is wrong here :wink:

File: css/style.css [hunk 2]
```diff
@@ -254,10 +254,14 @@ button:hover {
...
+	background-color: white;
 	min-height: initial;
 }
 
+.stack form {
+	width: 100%;
+}
+
 .stack h2 button,
 .stack .stack-actions {
```
Comment (juliushaertl) [inline css/style.css]: the margin should ideally only apply to the buttons, not to the container. otherwise the spacing is a bit to big.

Comment (artemanufrij) [inline css/style.css]: without `widht: 100%` it looks like this:
screenshot_20170519_185818

Comment (juliushaertl) [inline css/style.css]: I mean the rule below that :Wink:

.stack h2 button,
 .stack .stack-actions

Comment (artemanufrij) [inline css/style.css]: ah... ok... i guess we should fix it in an other branch. what do you thin?

**Pull request merged**